gem/templates/rwanda/core/section_page.html

```diff
@@ -58,17 +58,20 @@ <h3 class="article-teaser__title">
             <a class="call-to-action__read-more" href="{% pageurl article.specific %}">{% trans "Read more" %}</a>
             <ul class="teaser-stacklist-tags">
               {% load_tags_for_article article as tags %}
-              {% if tags %}
-                {% for tag in tags %}
-                {% if not tag.title.20 %}
-                  <li class="teaser-stacklist-tags__item">
-                    <a href="{% url 'tags_list' tag.slug %}" class="teaser-stacklist-tags__anchor">
-                      {{ tag.title }}
-                    </a>
-                  </li>
-                  {% endif %}
-                {% endfor %}
+              {% if settings.core.SiteSettings.enable_tag_navigation %}
```

Comment:
`{% if tags and settings.core.SiteSettings.enable_tag_navigation %}`
nothing major but the if block could be simplified